src/components/form/form.hbs

```diff
@@ -7,50 +7,85 @@
+  <input id="text-input-3" type="text"
+    class="{{@root.prefix}}--text-input{{#if light}} {{@root.prefix}}--text-input--light{{/if}}"
+    placeholder="Optional placeholder text">
+  <label for="text-input-3" class="{{@root.prefix}}--label">Text Input label</label>
   {{/if}}
 </div>
 <div class="{{@root.prefix}}--form-item">
   {{#if componentsX}}
-    <label for="text-area-2" class="{{@root.prefix}}--label">Text Area label</label>
-    <textarea id="text-area-2" class="{{@root.prefix}}--text-area{{#if light}} {{@root.prefix}}--text-area--light{{/if}}" rows="4" cols="50" placeholder="Placeholder text."></textarea>
+  <label for="text-area-2" class="{{@root.prefix}}--label">Text Area label</label>
+  <div class="{{prefix}}--form__helper-text">
+    Optional helper text here; if message is more than one line text should wrap (~100 character count maximum)
+  </div>
+  <textarea id="text-area-2" class="{{@root.prefix}}--text-area{{#if light}} {{@root.prefix}}--text-area--light{{/if}}"
+    rows="4" cols="50" placeholder="Placeholder text."></textarea>
   {{else}}
-    <textarea id="text-area-2" class="{{@root.prefix}}--text-area{{#if light}} {{@root.prefix}}--text-area--light{{/if}}" rows="4" cols="50" placeholder="Placeholder text."></textarea>
-    <label for="text-area-2" class="{{@root.prefix}}--label">Text Area label</label>
+  <textarea id="text-area-2" class="{{@root.prefix}}--text-area{{#if light}} {{@root.prefix}}--text-area--light{{/if}}"
+    rows="4" cols="50" placeholder="Placeholder text."></textarea>
+  <label for="text-area-2" class="{{@root.prefix}}--label">Text Area label</label>
   {{/if}}
 </div>
 <div class="{{@root.prefix}}--form-item">
-  <div class="{{@root.prefix}}--select{{#if light}} {{@root.prefix}}--select--light{{/if}}">
-    {{#if componentsX}}
-      <label for="select-id" class="{{@root.prefix}}--label">Select label</label>
+  <div
+    class="{{@root.prefix}}--select{{#if inline}} {{@root.prefix}}--select--inline{{/if}}{{#if light}} {{@root.prefix}}--select--light{{/if}}">
+    <label for="select-id" class="{{@root.prefix}}--label">Select label</label>
+    {{#unless inline}}
+    {{#if helperText}}
+    <div class="{{@root.prefix}}--form__helper-text">Optional helper text.</div>
     {{/if}}
-    <select id="select-id" class="{{@root.prefix}}--select-input">
-      {{#each selectItems}}
-        {{#if items}}
-          <optgroup class="{{@root.prefix}}--select-optgroup" label="{{label}}">
-            {{#each items}}
-              <option class="{{@root.prefix}}--select-option" value="{{value}}"{{#if disabled}} disabled{{/if}}{{#if selected}} selected{{/if}}{{#if hidden}} hidden{{/if}}>{{label}}</option>
-            {{/each}}
-          </optgroup>
-        {{else}}
-          <option class="{{@root.prefix}}--select-option" value="{{value}}"{{#if disabled}} disabled{{/if}}{{#if selected}} selected{{/if}}{{#if hidden}} hidden{{/if}}>{{label}}</option>
```

Comment:
Has `<select>` without `<optgroup>` been gone...? Though I see a bug in `master` where `select.config.js` not sending data for `<optgorup>` I see `{{#if}}` as well as `{{else}}` sections here has a solid meaning.

Reply:
I just updated the markup for the select component to what we have in `master`

Reply:
Thank you for the update @emyarod!